assets/js/app.js

```diff
@@ -122,3 +191,8 @@ function formatTime(datetime) {
     + ("0" + m.getUTCSeconds()).slice(-2);
 }
 
+// Sanitize string 
+function sanitizeString(str){
```

Comment:
interesting function. what lead you to create it? is there a requirement or comment in an issue somewhere?

Reply:
It's not necessary at all and I should remove it as it's not mentioned in the tutorial because it's unrelated. 
Just added it for nitpicking https://stackoverflow.com/questions/23187013/is-there-a-better-way-to-sanitize-input-with-javascript

Reply:
Nah, it's totally legit. Just add a comment with some context and an example of input and output. 💭

Reply:
Assign the PR back to me and I'll do it.

Reply:
Looks like this is where it's from: 
https://stackoverflow.com/questions/23187013/is-there-a-better-way-to-sanitize-input-with-javascript
<img width="1238" alt="image" src="https://user-images.githubusercontent.com/194400/219021621-c79ee872-dae3-4854-bb91-249cbf44098a.png">

I've added a comment to the code on the branch. 
Please avoid copy-pasting code (even in an example) 
without a comment to what problem it solves and where you found the solution. 
I know it's tempting to copy-paste code ... 
but when someone `else` has to _maintain_ it, comments are _essential_. 🙏